mockserver.js

```diff
@@ -60,11 +61,73 @@ var parse = function (content) {
         }
     });
 
-    body = bodyContent.join('\n');
+    body = applyHooks(bodyContent).join('\n');
```

Comment:
hey @pruski sorry to hear that! :(

Unfortunately, as you do, I also have limited time and resources -- so sometimes things will slip my mind :)
I think using a templating system will result in a better implementation, one that is more robust and easier to evolve -- that is why  I was suggesting to implement it that way, apologies for not bringing it up earlier :(

I would encourage you to give it a shot anyhow!